simtools/applications/simulate_showers_for_trigger_rates.py

```diff
@@ -1,232 +0,0 @@
-#!/usr/bin/python3
-r"""
```

Comment:
Completely removed this one?

Reply:
Yes - it simulates corsika showers. This is something simulate_prod can do - the functionality was duplicated.